ketcher-autotests/tests/Templates/Functional-Groups/Functional-Group-Tools/functional-group-tools.spec.ts

```diff
@@ -813,7 +813,6 @@ test.describe('Templates - Functional Group Tools3', () => {
     Description: Can attach copied Functional Group to atoms of structure
     Test not working proberly right now. Bug https://github.com/epam/ketcher/issues/2660
    */
-    test.fail();
```

Comment:
This change is not related to the sequence selection feature. Why we decided to do it here? If it really needed is comment above the test still valid?

Reply:
Because this test is valid, and this `test.fail();` will make the test fail. Sure, I will add comment.